lesson8/main.py

```diff
@@ -0,0 +1,73 @@
+import json
+from user import User
+
+
+def add_user(db, us):
+    db[us.name] = us.password
+
+
+def check_name(db):
+    while True:
+        username = input("Enter your username for a registration.\n")
+        if username in db.keys():
+            print("This username is already taken.")
+        elif 10 < len(username) or 2 > len(username):
```

Comment:
we check username here also.. lets move this check to the `User`.. we can use static or class method for this 😎